maykin_2fa/test/__init__.py

```diff
@@ -50,3 +50,6 @@ def get_valid_totp_token(user: AbstractBaseUser) -> str:
     if not isinstance(device, TOTPDevice):
         raise ValueError("The user's default device is not a TOTPDevice.")
     return _totp_str(device.bin_key)
+
+
+disable_admin_mfa = disable_mfa
```

Comment:
```suggestion
disable_admin_mfa = disable_mfa
"""
Test helper to disable MFA requirements in the admin.

Note that this is the same as :func:`disable_mfa`, because the `user.is_verified`
check is added via middleware which applies to the entire project and not just
the admin. However, this alias exists because maykin-2fa deliberately scopes
itself to managing access to the admin interface. Use the name that best conveys
your intent in your test cases.
"""
```